src/test/java/org/traccar/protocol/MictrackLowAltitudeFlightProtocolDecoderTest.java

```diff
@@ -0,0 +1,18 @@
+package org.traccar.protocol;
+
+import org.junit.Test;
+import org.traccar.ProtocolTest;
+
+public class MictrackLowAltitudeFlightProtocolDecoderTest extends ProtocolTest {
+
+    @Test
+    public void testDecode() throws Exception {
+
+        MictrackLowAltitudeFlightProtocolDecoder decoder = new MictrackLowAltitudeFlightProtocolDecoder(null);
+
+        verifyPositions(decoder, text("861108032038761$062232.00,A,2238.2832,N,11401.7381,E,0.01,309.62,95.0,131117"));
```

Comment:
Please format tests the same way all other tests are formatted.

Reply:
I added a position test that actually tests the expected value, I saw that was missing. Is that what you meant or can you explain more?

Reply:
Add new line like this:
```java
verifyPositions(decoder, text(
        "86110803...
```